src/Microsoft.AspNetCore.Rewrite/Internal/ApacheModRewrite/RuleBuilder.cs

```diff
@@ -66,13 +75,18 @@ public void AddRule(string rule)
             switch (input.ConditionType)
             {
                 case ConditionType.Regex:
+                    if (_regexTimeout == TimeSpan.Zero)
+                    {
+                        _regexTimeout = _uselowerRegexTimeouts ? TimeSpan.FromMilliseconds(1) : TimeSpan.FromSeconds(1);
```

Comment:
inline this on line 25?

Reply:
You mean doing:
```csharp
internal TimeSpan _regexTimeout =  _uselowerRegexTimeouts ? TimeSpan.FromMilliseconds(1) : TimeSpan.FromSeconds(1);
```

That wouldn't work as we are using a field to assign another field.

Reply:
Ah, then put it in the constructor.